modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/BSTValidator.java

```diff
@@ -75,12 +76,11 @@ public Credential validate(Credential credential, RequestData requestData) throw
     public boolean authenticate(String binarySecurityToken) throws
```

Comment:
As the method returns a boolean, better to name it as something like "isAuthenticated"